shared/js/background/message-handlers.js

```diff
@@ -33,11 +33,8 @@ export function getBrowser () {
     return browserName
 }
 
-export function firePixel (fireArgs) {
-    if (fireArgs.constructor !== Array) {
```

Comment:
any idea why we had this check before?

Reply:
`firePixel('eph')` was used in a few places for example and it also supported the `firePixel(['eph', ...])`